assets/src/edit-story/components/canvas/pagemenu/index.js

```diff
@@ -186,7 +186,7 @@ function PageMenu() {
           <WithTooltip title={__('Duplicate page', 'web-stories')}>
             <Icon
               onClick={handleDuplicatePage}
-              aria-label={__('Dupliccate Page', 'web-stories')}
+              aria-label={__('Duplicate Page', 'web-stories')}
```

Comment:
Ouch! Good catch. Since it's unrelated, perhaps move it into a separate PR?

Reply:
Added: https://github.com/google/web-stories-wp/pull/2905